io/net/virt-net/network_virtualization.py

```diff
@@ -35,6 +35,9 @@
 from avocado.utils.network.hosts import LocalHost
 from avocado.utils.ssh import Session
 from avocado.utils import wait
+import pexpect
+from pexpect import pxssh
```

Comment:
can we have readme or class comment as this need to be installed before running test